src/core_modules/capture-ui/FlatList/flatList.types.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as React from 'react';
```

Comment:
I would only import the `type Element` in this case, but that's only a suggestion :) 
```suggestion
import { type Element } from 'react';
```